v3/examples/theme/assets/style.css

```
@@ -0,0 +1,59 @@
+
+.btn {
+    padding: 5px 10px;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+    width: 100vw;
+    height: 100vh;
+    gap: 1px;
```

Biome (2.4.6)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)